### ISSUE TITLE
Why are position/width/height styles needed when plainSprite is used

### ISSUE THREAD

**Issue by VladShcherbin** [kisenka/svg-sprite-loader#232]
Hey,

I'm using the loader to extract `sprite.svg` file with `plainSprite` option. I'm referencing svg files later like this one:

```html
<svg viewBox="0 0 32 32">
  <use xlinkHref="sprite.svg#test" />
</svg>
```

The generated sprite file starts with `<svg style="position: absolute; width: 0; height: 0" ...`. Why do I need this styles when I'm using external file. Am I missing something?

**Comment by kisenka** [comment]
Hola! Thanks for reporting, it's a bug, fixed in 3.6.1

**Comment by VladShcherbin** [comment]
@kisenka holy crap, this was blazing fast. Thank you for such a great loader! ❤️
